publish_bad_checksum: false # mrs_serial will publish messages with incorrect checksums
